src/components/QuizData.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import '../QuizData.css';
+
+export default class QuizData extends Component {
+  constructor() {
+    super();
+    this.state = {
+      quizzes: [],
+      questions: [],
+    };
+  }
+
+  componentDidMount() {
+    axios.get('/quizzes')
+      .then((response) => {
+        this.setState({
+          quizzes: response.data.quizzes,
+          questions: response.data.quizzes[0].questions
+        });
+      })
+      .catch((error) => {
+        console.log(error);
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        {this.state.quizzes.map((quiz) => {
+          return (
+            <section key={quiz.id}>
+              <h1 className="pageTitle">{quiz.title}</h1>
+            </section>
+          );
+        })}
+        {this.state.questions.map((quiz) => {
+          return (
+            <section key={quiz.id} className="questionContainer">
+              <h3 className="questionTitle">{quiz.title}</h3>
+              {quiz.answers.map((quiz, unique) =>
+                <ul key={unique}>
+                  <li>{quiz.title}</li>
+                </ul>
+              )}
+            </section>
```

Comment:
We might want to break up this block of HTML we're returning by creating some additional components. If we add a Question component and an Answer component, we could render something like the following:

questions.map((question, index) =>
    <Question title={question.title} key={question.id}/>
    (question.answers).map(answer =>
        <Answer title={answer.title} />
    )
(That is not going to be the exact syntax, and I'm sure it will break in all sorts of ways, but that's what we should be going for here.)